config/groups.yaml

```diff
@@ -639,3 +639,7 @@
     - "2575138"
     - "2522944"
   id: 190041
+- name: "Group LuNiJo"
+- members:
+    - "2879998"
+    - "2885512"
```

Comment:
Just to check, are you happy to have only two members in your group? The normal minimum is 3, but I am happy with this if you are.